content/tutorials/guides/how-to-get-pim-product-information.md

```diff
@@ -215,6 +229,48 @@ function fetchProducts(): array
     return array_merge(...$products);
 }
 ```
+```javascript [activate:nodeJS]
+
+async function fetchProducts() {
+  const locales = await getlocales();
```

Comment:
I guess you retrieve locales and family codes from the storage. Add comments to precise it